### PR TITLE
feat(blockstore): Implement LruBlockstore

### PR DESCRIPTION
**Pull request by oblique**

### CODE REVIEW

File: blockstore/src/lru_blockstore.rs [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+            cache: Mutex::new(LruCache::new(capacity)),
+        }
+    }
+}
+
+#[cfg_attr(not(docs_rs), async_trait::async_trait)]
+impl<const MAX_MULTIHASH_SIZE: usize> Blockstore for LruBlockstore<MAX_MULTIHASH_SIZE> {
+    async fn get<const S: usize>(&self, cid: &CidGeneric<S>) -> Result<Option<Vec<u8>>> {
+        let cid = convert_cid(cid)?;
+        let mut cache = self.cache.lock().unwrap();
```
Comment (bishopcheckmate) [inline blockstore/src/lru_blockstore.rs:26]: can we use expect with some note why is it okay to panic?

Comment (zvolin) [inline blockstore/src/lru_blockstore.rs:26]: can we use expects there?

**Review by zvolin — APPROVED** [review]
looks good, left a nit

**Review by zvolin — APPROVED** [review]

**Review by fl0rek — APPROVED** [review]
nice 🎸

**Pull request merged**